deis-workflow-manager/manifests/deis-workflow-manager-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  type: LoadBalancer
+  selector:
+    app: deis-workflow-manager
+  ports:
+    - name: https
+      port: 8443
+      targetPort: 8443
```

Comment:
Curious about the selection of 8443.  The fact that it ends in 443 _hints_ that the protocol is HTTPS and that we're just not using 443 because it's a privileged port.  What's listening within the container?  (I'm not sure what the tech stack is.)  Is any SSL involved?  If so, I would again say it might be better for the router to handle that stuff.  If not, then some port like 8000, 8080, or 8888 might be better because it doesn't hint at HTTPS.